app/test/symbolManagement.test.ts

```
@@ -1,102 +1,162 @@
+jest.mock('@nozbe/watermelondb/adapters/lokijs', () =>
+  jest.requireActual('@nozbe/watermelondb/adapters/lokijs'),
+);
+jest.mock('../db', () => {
+  const { Database } = jest.requireActual('@nozbe/watermelondb');
+  const LokiJSAdapter =
+    jest.requireActual('@nozbe/watermelondb/adapters/lokijs').default;
+  const { mySchema } = jest.requireActual('../db/schema');
+  const { migrations } = jest.requireActual('../db/migrations');
+  const {
+    Profile,
+    Symbol,
```

Biome (2.1.2)

[error] Do not shadow the global "Symbol" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)